Libraries/Utilities/StringExtensions.cs

```diff
@@ -56,6 +66,6 @@ public static class StringExtensions
             this string value,
             string separator,
             StringSplitOptions stringSplitOptions = StringSplitOptions.None) =>
-            value?.Split(new[] { separator }, stringSplitOptions) ?? Array.Empty<string>();
+            value?.Split(new[] { separator }, stringSplitOptions) ?? new string[] { };

```

Comment:
I don't know what the default array size is during construction, but returning `new string[0]` will definitely keep it to 0:
```suggestion
            value?.Split(new[] { separator }, stringSplitOptions) ?? new string[0];

```

Reply:
Also please add a comment above this line explaining why we can't use `Array.Empty<string>()` here.